### ISSUE TITLE
Structure of the repository

### ISSUE THREAD

**Issue by Wrent** [cngroupdk/CNU#3]
I think we should discuss how we want to structure the repository to prevent mess of lot of different files.

Maybe we can have some technology related lists on the root level and then we can have folders to store more materials? But we also need to decide how to handle cross-technology concerns (GIT, REST...)

**Comment by lenkavon** [comment]
Definitely, we need to agree. There is more way how to structure - have one overview and then technology folders or department folders.

For now we can just aggregate what we have and when we will have better overview what the material will be about we can give it a structure.
